mantle/cmd/ore/aliyun/make-public.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aliyun
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdPublic = &cobra.Command{
+		Use:   "make-public <image:region...>",
+		Short: "Mark images on aliyun publicly available",
+		Long:  `Mark one or more images on aliyun publicly available`,
+		RunE:  makePublic,
+
+		SilenceUsage: true,
+	}
+)
+
+func init() {
+	Aliyun.AddCommand(cmdPublic)
+}
+
+func makePublic(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		fmt.Fprintf(os.Stderr, "Specify one ore more image:region pairs.\n")
+		os.Exit(2)
```

Comment:
Since this is using `RunE`, you should be able to directly return errors instead of printing and calling `os.Exit()`.

Reply:
I was matching what was done in `create-image.go` and `copy-image.go`, though it looks like `aliyun.go` does do `return fmt.Errorf()`, so there is some mis-match in this code